validphys2/src/validphys/theorycovariance/output.py

```diff
@@ -90,8 +90,8 @@ def matrix_plot_labels(df):
 def plot_covmat_heatmap(covmat, title):
     """Matrix plot of a covariance matrix."""
     df = covmat
-    df.sort_index(0, inplace=True)
-    df.sort_index(1, inplace=True)
+    df.sort_index(axis=0, inplace=True)
+    df.sort_index(axis=1, inplace=True)
```

Comment:
Is now `axis` mandatory?

Reply:
for mer it was not working without see also: https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.sort_index.html

And I think it's also backward compatible, so to me it seems okay to merge, otherwise I can revert.